recipes/diligent-tools/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,6 @@
+#include "Common/interface/RefCntAutoPtr.hpp"
+
+int main()
+{
+  return 0;
```

Comment:
it would be better if it could at least construct something, so we check it actually links with library

Reply:
done